src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -85,22 +87,50 @@ public boolean canCheck( final Object toVerify ) {
 
 	@Override
 	public Set<RootElement> convert( final Object toVerify ) {
+		final RemoteWebElement webElement = retrieveWebElement( toVerify );
+		if ( webElement != null ) {
+			return convertWebElement( webElement );
+		}
+		final WebDriver webDriver = retrieveWebDriver( unwrapImplicitDriver( toVerify ) );
+		if ( webDriver != null ) {
+			return convertWebDriver( webDriver );
+		}
+		throw new IllegalArgumentException( "Cannot convert objects of type '" + toVerify.getClass().getName() + "'." );
+	}
+
+	private RemoteWebElement retrieveWebElement( final Object toVerify ) {
 		if ( SeleniumWrapperUtil.isWrapper( WrapperOf.ELEMENT, toVerify ) ) {
-			return convert( SeleniumWrapperUtil.getWrapped( WrapperOf.ELEMENT, toVerify ) );
+			return retrieveWebElement( SeleniumWrapperUtil.getWrapped( WrapperOf.ELEMENT, toVerify ) );
 		}
 		if ( toVerify instanceof RemoteWebElement ) {
-			return convertWebElement( (RemoteWebElement) toVerify );
+			return (RemoteWebElement) toVerify;
+		}
+		return null;
+	}
+
+	private Object unwrapImplicitDriver( final Object toVerify ) {
+		if ( toVerify instanceof AutocheckingRecheckDriver ) {
+			throw new UnsupportedOperationException( String.format(
+					"Mixing implicit checking used by '%s' and explicit checking with 'Recheck#check' is not supported.",
```

Comment:
This error message is not "actionable". Can we improve the error message, i.e. tell the user how to proceed from here? Examples could be a link to the docs or FAQs or similar.

Reply:
Alternative error message: "The AutocheckingRecheckDriver does automatically check after each action, therefore no explicit call to re.recheck() is needed. For more information, please have a look at <link to docs>."

Reply:
Why didn't I think of this? This is not a message for us but for the user, therefore it should be actionable. I addressed this with a link to the documentation. Please be aware that the above mentioned docs changes must be merged in order for the link to get valid.